src/components/app/app.ts

```diff
@@ -0,0 +1,43 @@
+import AppController from '../controller/controller';
+import { AppView } from '../view/appView';
+import { Data } from '../view/appView';
+import ChangeColor from './changeColors';
+
+class App {
+    private controller = new AppController();
+    private view = new AppView();
+    private changeColor = new ChangeColor();
+
+    start() {
+        const sources1 = document.querySelector('.sources') as HTMLElement;
+        sources1.addEventListener('click', (e) => this.controller.getNews(e, (data?: Data) => {
+            if (data) {
+                this.view.drawNews(data)
+            }
+        }));
+        this.controller.getSources((data?: Data) => {
+            if (data) {
+                this.view.drawSources(data)
+            }
+        });
+
+        const button1 = document.querySelector('button.light') as Element;
```

Comment:
```suggestion
        const button1 = document.querySelector('button.light') as HTMLElement;
```
Here and below. Use more specific types, please 

And better next
```suggestion
        const button1: HTMLElement | null = document.querySelector('button.light');
```

Reply:
Added such a variant: const button1 = <HTMLButtonElement>document.querySelector('button.light');